src/pages/Base.tsx

```diff
@@ -6,7 +6,7 @@ const Base = (props: React.PropsWithChildren<Record<never, never>>) => {
     return (
         <Container maxW="container.xl">
             <Navbar />
-            <React.Suspense fallback={<></>}>{props.children}</React.Suspense>
```

Comment:
@alexnguyenn I thought it would have made sense to have the lazy loading at the page level because we don't need a page to load until we actually navigate to it. Are al routes loaded on page load? or there is a second layer of lazy loading in the routes?

Also not sure how all of this plays differently between the React DOM and the actual DOM

Reply:
And I also thought loading the navbar first and then lazily loading the rest would have given the user confidence, by first showing the navbar

Reply:
I updated it a little bit in e711d6e - you might have missed that:
https://github.com/latticesurgery-com/web-ui/blob/e711d6e317ef94d491daf43c092f72d44818f135/src/index.tsx#L21-L31

So we are lazy-loading the content of each page (the `props.children` passing into `Base`), but not the `Base` component. This means that components in `Base` like `Navbar` (or maybe later `Footer`) will not be lazy-loaded. So what you mentioned above is already implemented - tho I might have missed something.

Reply:
You are right Base is outside of the routes. That is desired behavior